lib/ElementHandle.js

```diff
@@ -103,6 +103,19 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Object}
+   */
+  async boundingBox() {
+    const boxModel = await this._client.send('DOM.getBoxModel', { objectId: this._remoteObject.objectId });
```

Comment:
The bounding box of the element could be partially off-screen. The element might also not be scrolled into view. something to consider when taking screenshots. If you look above at `_visibleCenter`, we use `scrollIntoViewIfNeeded` and clip based on the window bounds. Because we are already in JavaScript, we use getBoundingClientRect and avoid introducing `DOM.getBoxModel`.

I don't expect `element.boundingBox()` to scroll the element into view, but I can imagine that it would be intended if boundingBox is used to take screenshots.